### PR TITLE
DOCS-11130-adding note about testing app

### PR DESCRIPTION
**Pull request by IsaacEldridge**
adding note about testing app and remove unnecessary table captions

### CODE REVIEW

File: modules/ROOT/pages/deploy-resource-allocation.adoc [hunk 1]
```diff
@@ -87,10 +87,9 @@ The available heap memory is approximately half of the total memory allocated to
 
 == Application Startup Times
 
-The startup time for a Mule applications is correlated to the total number of vCPU cores the application has access to:
+The startup time for a Mule application is correlated to the total number of vCPU cores the application has access to. The following lists approximate startup times for a single proxy Mule application performing simple processing on a payload:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/deploy-resource-allocation.adoc]: @vli63127 here's the note about the testing app

Comment (vli63127) [reply modules/ROOT/pages/deploy-resource-allocation.adoc]: I would reword it as "for a simple Mule proxy application performing processing on a payload".

**Review by sarjeet2013 — APPROVED** [review]

**Review by valkyrie69 — APPROVED** [review]
